web/app/js/components/ServiceMesh.jsx

```diff
@@ -27,6 +25,58 @@ const serviceMeshDetailsColumns = [
     className: "numeric"
   }
 ];
+
+const barColor = percentMeshed => {
+  if (percentMeshed <= 0) {
```

Comment:
related to other percent comment, how does `percentMeshed <= 0` happen? there's code further down that does `row.meshedPercent.get() > 0.9 ? "good" : "neutral";`. how does this relate to that?

Reply:
A percent meshed < 0 can happen if there are 0 total pods (denominator is 0, so the percentage helper returns -1 as the decimal and "N/A" as the pretty formatted).

The barColor is independent of this; it's just deciding whether to color the bar green or grey (and in the future or red) based on the percentage of pods that have already been meshed.